react/features/conference/components/Conference.native.js

```diff
@@ -388,12 +392,16 @@ function _mapDispatchToProps(dispatch) {
          * Dispatches actions to create the desired local tracks and for
          * connecting to the conference.
          *
+         * FIXME: Do these have to be here? Shouldn't this be in a middleware
+         * instead?
+         *
          * @private
          * @returns {void}
          */
         _onConnect() {
             dispatch(createDesiredLocalTracks());
             dispatch(connect());
+            dispatch(clearChat());
```

Comment:
This doesn't belong here. The chat should be cleared when the conference is left, not when it's joined.